projects/plugins/jetpack/modules/scan/class-admin-sidebar-link.php

```diff
@@ -73,7 +74,7 @@ public function maybe_add_admin_link() {
 			$menu_label = __( 'Scan', 'jetpack' );
 			$url        = Redirect::get_url( 'calypso-scanner' );
 		} elseif ( ! $has_scan && $show_backup ) {
-			$menu_label = __( 'Backup', 'jetpack' );
+			$menu_label = __( 'VaultPress Backup', 'jetpack' );
 		} else {
 			// Will be both, as the code won't get this far if neither is true (see should_show_link()).
 			$menu_label = __( 'Backup & Scan', 'jetpack' );
```

Comment:
You may want to also add VaultPress here for consistency, but if so the name may be pretty long for the menu.

Reply:
highlighted the wrong line, I was referring to line 79/80

Reply:
Yeah, the external link icon is jumping to the next line:
<img width="161" alt="image" src="https://user-images.githubusercontent.com/1488641/215617469-472a1f30-9ab4-4e65-8f3a-481e755049b8.png">

Reply:
I personally don't like how it looks like, would prefer to get rid of the external link icon as I think it would be better to keep the branding. What do you think? @nunyvega @miguelxpn

Reply:
Removing the space before the icon make it looks better:
<img width="161" alt="image" src="https://user-images.githubusercontent.com/1488641/215630737-24fce3e4-685a-4b73-8e0d-0a638f8164d0.png">

Reply:
> would prefer to get rid of the external link icon 

I wouldn't do this. We want to be clear whenever we are taking a user outside of their site.

Reply:
> Removing the space before the icon make it looks better:

I like how this looks too!
If for some reason it still doesn't fit due to the long name, another approach can be to add Backup and Scan in two separate lines;